### DISCUSSION TITLE
Following option dialog never find any useful usage when use with GNOME + Wayland.

### DISCUSSION THREAD

**Discussion by zw963** [flameshot-org/flameshot#2360] [Q&A] [answered]
Anyone can help on how to use following option dialog?

screenshot1
![image](https://user-images.githubusercontent.com/549126/152353755-e82c5353-5e75-43dc-a0cd-e458b45a8bf3.png)

As we know, following dialog always pop up when use with GNOME + Wayland.

screenshot2
![image](https://user-images.githubusercontent.com/549126/152354763-dc4c1316-533b-40b0-8b54-31114fb82ce2.png)

we have to click `Share` Button, then select and edit the screenshot. When i click the `Options` button, i can get previous 
dialog, but, what is the usage for screenshot1 ? i never find any useful usage after click this option button, then, click `take screenshot`, because we have to take screenshot one more time after click `take screenshot`, right?

Anyone can give me some clue about how to use this?

Thank you.

**Comment by mmahmoudian** [comment, marked as answer]
This is Gnome Wayland thing. That window you see if from them which is asking for permission to share the screenshot with Flameshot (the screenshot of your full screen is taken by Gnome and then shared to other softwares like Flameshot).

You can read more about it in https://flameshot.org/docs/guide/wayland-help/

> because we have to take screenshot one more time after click take screenshot, right?

No, it is the same screenshot that Gnome takes that is shared with Flameshot and Flameshot opens that yo let you annotate it.

> Anyone can give me some clue about how to use this?

If you want to use Flameshot, always press "share" and the  use Flameshot normally.

**Reply by zw963** [reply]
> If you want to use Flameshot, always press "share" and the use Flameshot normally.

Yes, thank you, in fact, i know it before, what i means is, except click `share` button, there is no any meanful
things to do when click `option` button when use with GNOME + Wayland, why we still show that `options` button?

What i said is, The options button can be remove, right? we only need click `share` to get gnome authorized to do screenshot, right?

**Reply by mmahmoudian** [reply]
You are right about the options button. That is something that the Gnome devs should answer/address. That window is Gnome's window and we at Flameshot cannot change it.
